core/domain/suggestion_services.py

```diff
@@ -1125,22 +1120,21 @@ def get_reviewable_translation_suggestions_for_single_exp(
     """
     contribution_rights = user_services.get_user_contribution_rights(
         user_id)
+    user_settings = user_services.get_user_settings(user_id)
+    prefered_language_code = user_settings.preferred_translation_language_code
     language_codes = (
         contribution_rights.can_review_translation_for_language_codes)
 
-    if language is not None:
-        language_codes = [language] if language in language_codes else []
-
-    # The user cannot review any translations, so return early.
-    if len(language_codes) == 0:
+    # The user cannot review any translations or in preferred lang,
```

Comment:
I don't really understand this comment. Could you please explain it more clearly? Also, don't use abbreviations like "lang", write full sentences.